config/manager/manager.yaml

```diff
@@ -47,7 +47,7 @@ spec:
               name: RELATED_IMAGE_sdc
             - value: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0
               name: RELATED_IMAGE_karavi-authorization-proxy
-            - value: quay.io/dell/container-storage-modules/dell-csi-replicator:v1.12.0
+            - value: quay.io/dell/container-storage-modules/dell-csi-replicator:v1.10.0
```

Comment:
any reason for downgrading the version?

Reply:
@shefali-malhotra The current available version is 1.9.0 and next release version should be 1.10.0
@satyakonduri

Reply:
Yes, @ashleyvjoy, the upcoming version is 1.10.0.